src/main/java/frc/robot/Constants.java

```diff
@@ -14,7 +14,7 @@
 */
 public final class Constants {
     public static final int KITBOT_ROLLER_ID = 5; //TODO: Verify ID
-
+    public static final int MAIN_BOT_CORAL_ID = 7;
```

Comment:
Presumably this is for the Coral grabber on the EndEffector. Make the name more specific.